Proposals/xgov-188.md

```diff
@@ -0,0 +1,152 @@
+
+## Future Blueprint
+PiPhi Network plans to reward users for developing or integrating IoT devices with home automation to enable further distribution of the software making both PiPhi Network and Algorand a household name in the most literal sense. The strategic goal for PiPhi is to serve as the premier platform to aggregate data that spans the spectrum of IoT data from enterprise supervisory control and data acquisition (SCADA) to entry level tinkerers and DIY enthusiasts. To achieve this vision, PiPhi plans to implement an open-source funding program that allows developers inside or outside of the community to develop plugins for the PiPhi Network platform and receive rewards for plugin usage and installations. Imagine a Home Assistant platform with tokenized incentives to build. We believe this could be an innovative platform in becoming a truly community driven project in the DePIN/IoT web3 space.
+
+## Benefits for the community
+https://chibisafe.piphi.network/Q9QBXgJ38Set.png 
+
+Beyond a customizable and centralized smart home visualization platform, cross platform compatibility for hardware distributed projects and partnerships, the Algorand community will benefit from the additional interest of developers. PiPhi Network software is a developer focused incentive framework. Developers are compensated through the PIPHIN token for participating in further development of the Algorand ecosystem and PiPhi network by writing code, adding new sensors, and/or features to the software.  
+
+## Additional information
+Funding Allocation 
+
+  
+
+Tier 1: Full Funding 
+
+  
+
+PiPhi requests $155,000 (645833 ALGO) in outside funding to support operations and development for the next 12 months. This amount will sustain the salaries of two full-time hires.  
+
+  
+
+Employee 1 ($77,500/Yr): Responsible for PiPhi development, coding, testing, error identification and error patching. 
+
+  
+
+Employee 2 ($77,500/Yr): Responsible for PiPhi development, coding, testing, error identification and error patching. 
+
+
+Tier 2: Partial Funding 
+
+  
+
+PiPhi requests $115,000 (479,166 ALGO) in outside funding to support operations for the next 12 months. This amount will sustain the salary of one full-time and one part-time hire. Part-time work is defined as work that does not exceed 20 hours per standard work week. Hourly wage is approximately $38/Hour. Timelines for milestone completion will likely exceed 12 months and be in the range of 16 to 18 months. This could require further cash injections to cover operating and development costs. 
+
+  
+
+Employee 1 ($75,000/Yr): Responsible for customer service, resolving support tickets, and development. 
+
+  
+
+Employee 2 ($40,000): Responsible for customer service and resolving support tickets. 
+
+
```

Comment:
Grants are only paid when all is delivered. [Please read ARC-34](https://arc.algorand.foundation/ARCs/arc-0034) to ensure you understand the rules.